### DISCUSSION TITLE
How to define non-renal clearance or total body clearance in PKSim

### DISCUSSION THREAD

**Discussion by Xiaoyang-svg** [Open-Systems-Pharmacology/Forum#1321] [How-Tos]
Hi everyone! 

I have a couple of questions on defining non-renal clearance in PkSim. My drug have non-renal clearance with <1% of drug excreted unchanged in urine, and I don't know how to define a non-renal route of elimination process. Since there is far from enough information on the pharmacokinetic properties of this drug reported in the literature, I also wonder if there is a way to use the total body clearance for the elimination processes? 

If you have any solutions, please don't hesitate to reply.

Thank you in advance.

**Comment by StephanSchaller** [comment]
Hi @Xiaoyang-svg, here is the description of how to implement metabolism (e.g. liver clearance):
https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-compounds-definition-and-work-flow#metabolism

Which is your drug (publicly known compound if there is literature on it?)?

**Reply by Xiaoyang-svg** [reply]
Thanks a lot for your reply. 

I'm working on polymyxin B, which is considered a last-line therapy for MDR gram-negative bacilli. The main problem is that its elimination process has so far not been established. Most researchers believe it has nonrenal clearance, but the exact mechanism remains unclear. So I want to know how to describe its elimination process in PKSim.

**Reply by StephanSchaller** [reply]
There are a number of ways to account for non-renal elimination / metabolism in PK-Sim.

You can, e.g., define a metabolizing enzyme with a specific tissue expression profile (also in plasma to account for a compound transformation or instability) or a total hepatic clearance. The link above should give you an idea how to do this. Alternatively, I can point you to our training material:
- for academia (free): https://learn-esqlabs.talentlms.com
- for industry (subscription): https://training-esqlabs.talentlms.com

**Reply by Xiaoyang-svg** [reply]
Thank a lot for your feedback. I actually read a similar paper of  a PBPK model for polymyxin E (doi: 10.1007/s40262-022-01161-y.), which was also constructed in pksim and defined renal and nonrenal clearance. So, I orginally wanted to refer to this literature. In addition, may I ask how to define renal abosorption in pksim?

Thanks advance.

(https://user-images.githubusercontent.com/102672543/195993423-4c1c9475-e710-4fc0-8339-1e7f6b525e57.png)
